specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2022-06-01/managedClusters.json

```diff
@@ -3001,6 +3002,10 @@
               {
                 "value": "kubenet",
                 "description": "Use the Kubenet network plugin. See [Kubenet (basic) networking](https://docs.microsoft.com/azure/aks/concepts-network#kubenet-basic-networking) for more information."
+              },
+              {
+                "value": "none",
+                "description": "Use no CNI plugin pre-installed. See [BYO CNI](https://docs.microsoft.com/en-us/azure/aks/use-byo-cni) for more information."
```

Comment:
minor: Consider rephrasing as: "Do not use a pre-installed CNI" or maybe "No CNI plugin is pre-installed" (and then the doc link)

Reply:
Changed, thanks!